runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -145,7 +145,9 @@ private void overrideSprites()
 			}
 		}
 
-		client.setSpriteOverrides(overrides);
+		client.getWidgetSpriteCache().reset();
+		client.getSpriteOverrides().clear();
```

Comment:
would you really want to clear the entire map if someone else is using sprite overrides?

Reply:
I mostly just made the change to do what it was doing till now, but now there is possibility to alter this to make it work with more plugins